README.md

```diff
@@ -1,2 +1,72 @@
-# code-review-role
-This is the process of code review
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio - Jorum Odiemo
+
+> Creating a skeleton mobile website of my portfolio
+> 
+
+
+## Built With
+
+- HTML & CSS3
+- Flexbox and Grid
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
```

Comment:
The comments are good. Don't you think it would have been good to add that a prerequisite project was the first project you do n creating the Mobile Skeleton Version? Just my thought 😁